lumera/chain.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "lumera",
+  "chain_type": "cosmos",
+  "chain_id": "lumera-mainnet-1",
+  "pretty_name": "Lumera",
+  "website": "https://lumera.protocol",
+  "network_type": "mainnet",
+  "status": "upcoming",
+  "bech32_prefix": "lumera",
+  "bech32_config": {
+    "bech32PrefixAccAddr": "lumera",
+    "bech32PrefixAccPub": "lumerapub",
+    "bech32PrefixValAddr": "lumeravaloper",
+    "bech32PrefixValPub": "lumeravaloperpub",
+    "bech32PrefixConsAddr": "lumeravalcons",
+    "bech32PrefixConsPub": "lumeravalconspub"
+  },
+  "daemon_name": "lumerad",
+  "node_home": "$HOME/.lumera",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ulume",
+        "fixed_min_gas_price": 0.025,
+        "low_gas_price": 0.025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ulume"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/LumeraProtocol/lumera",
+    "tag": "v1.0.0",
+    "recommended_version": "v1.0.0",
+    "language": {
+      "type": "go",
+      "version": "v1.24"
+    },
+    "binaries": {
+      "linux/amd64": "https://github.com/LumeraProtocol/lumera/releases"
```

Comment:
Path: binaries.linux/amd64, Reason: Value mismatch, In "codebase": https://github.com/LumeraProtocol/lumera/releases, In currentVersion: https://github.com/LumeraProtocol/lumera/releases/download/

Reply:
The values need to match between the versions file and the chain file.

Reply:
we don't yet have binaries for mainnet, it's "upcoming". Should i just remove both "binaries" from both `chain.json` and `versions.json`?

Reply:
yes